package.json

```diff
@@ -35,8 +35,9 @@
     "custom-logger": "0.3.1",
     "inquirer": "0.10.0",
     "isemail": "1.2.0",
+    "lodash.isempty": "4.4.0",
+    "lodash.isnil": "4.0.0",
```

Comment:
Same comment above could apply to this dependency

Reply:
Removed `isnil` but `isempty` is still there because it contains a bit more logic.